### PR TITLE
URL import docs

### PR DESCRIPTION
**Pull request by matkuliak**
## Summary of the changes / Why this is an improvement
- URL imports, pretty short for now, not sure what additional info could be useful, feel free to comment
- Couple of pics updates, didn't feel like making a separate PR

## Checklist

 - [x] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

File: docs/overview.rst [hunk 1]
```diff
@@ -339,6 +339,36 @@ environment.
...
+Gzip compressed files are also supported.
+
+.. NOTE::
+
+    - CSV files must have a header, otherwise the first row will be used as
+      headers.
+    - For JSON documents, each line must be a JSON document (arrays not
+      supported)
+    - For JSON documents, only "document-per-line" is supported -> 
+      https://jsonlines.org/
```
Comment (SStorm) [inline docs/overview.rst:370]: Maybe also make jsonlines.org a link?
```suggestion
    - For JSON files, only "document-per-line" is supported, as defined in https://jsonlines.org/
    - Additionally for JSON files, each line must be a JSON document (arrays not
      supported)
```

File: docs/overview.rst [hunk 2]
```diff
@@ -339,6 +339,36 @@ environment.
...
+To import data, simply fill out the URL, name of the table which will be
+created and populated with your data, data format, and compression.
+
+Following data formats are supported:
+
+- CSV
+- JSON
+- Parquet
+
+Gzip compressed files are also supported.
```
Comment (SStorm) [inline docs/overview.rst:361]: Might also be worth mentioning that:
```
A table will be created for you if it does not exist. Please note that in this case the importer will attempt to guess the data types of each column, and might do so sub-optimally. 
```

File: docs/overview.rst [hunk 3]
```diff
@@ -339,6 +339,36 @@ environment.
...
+
+In the Data tab, users can conveniently import data from a URL.
+
+.. image:: _assets/img/cluster-data-tab.png
+   :alt: Cloud Console cluster data tab
+
+To import data, simply fill out the URL, name of the table which will be
+created and populated with your data, data format, and compression.
+
+Following data formats are supported:
```
Comment (SStorm) [inline docs/overview.rst:355]:
```suggestion
The following data formats are supported:
```

File: docs/overview.rst [hunk 4]
```diff
@@ -339,6 +339,36 @@ environment.
...
+Data
+----
+
+In the Data tab, users can conveniently import data from a URL.
+
+.. image:: _assets/img/cluster-data-tab.png
+   :alt: Cloud Console cluster data tab
+
+To import data, simply fill out the URL, name of the table which will be
+created and populated with your data, data format, and compression.
```
Comment (SStorm) [inline docs/overview.rst:353]:
```suggestion
created and populated with your data, data format, and whether it is compressed.
```

**Review by SStorm — APPROVED** [review]
Good start :+1:

File: docs/overview.rst [hunk 5]
```diff
@@ -64,8 +64,8 @@ Multiple clusters and users can exist in a organization at any moment. For
 first-time users, an organization called "My organization" is automatically 
 created upon first login.
 
-To see list of all the clusters in your organization, go to the Account
-tab in your left-hand menu:
+To see list of all the organizations your account is associated with, go to
```
Comment (kojinkai) [inline docs/overview.rst:67]:
```suggestion
To see a list of all the organizations you have acesss to, go to 

```

**Pull request merged**